src/plugins/fts-squat/fts-backend-squat.c

```diff
@@ -137,6 +147,9 @@ fts_backend_squat_set_box(struct squat_fts_backend *backend,
 	if (backend->full_len != 0)
 		squat_trie_set_full_len(backend->trie, backend->full_len);
 	backend->box = box;
+	ret = squat_trie_open(backend->trie);
+	if (ret < 0)
+		return -1;
```

Comment:
It's done now and I see it's been picked up automatically by the pull request. Thanks for pointing it out.